### PR TITLE
add full description of frame fields

### LINKED ISSUE #93: Better specification of Frame fields
The textual description of HTTP/2 and HTTP/3 frame fields in not comprehensive, so we should fix this. For example, as @ianswett highlighted, the HTTP/2 PRIORITY_UPDATE frame diagram declares the `R` bit but doesn't specifiy what that means, the answer is a direct copy of RFC 7540 section 4.1

```
    R: A reserved 1-bit field.  The semantics of this bit are undefined,
      and the bit MUST remain unset (0x0) when sending and MUST be
      ignored when receiving.
```

### PR DESCRIPTION
**Pull request by LPardue**
fixes #93

### CODE REVIEW

File: draft-kazuho-httpbis-priority.md [hunk 1]
```diff
@@ -449,6 +449,20 @@ frame header MUST be zero (0x0).
 ~~~
 {: #fig-h2-reprioritization-frame title="HTTP/2 PRIORITY_UPDATE Frame Payload"}
 
+The PRIORITY_UPDATE frame payload has the following fields:
+
+R: 
+: A reserved 1-bit field. The semantics of this bit are undefined,
+and the bit MUST remain unset (0x0) when sending and MUST be
```
Comment (ianswett) [inline draft-kazuho-httpbis-priority.md]: I think ideally this would all be indented, and below

File: draft-kazuho-httpbis-priority.md [hunk 2]
```diff
@@ -449,6 +449,20 @@ frame header MUST be zero (0x0).
...
+ignored when receiving.
+
+Stream ID:
+: A 31-bit stream identifier for the stream that is the target of the priority 
+update.
+
+Priority Field Value:
+: The priority update value in ASCII text.
```
Comment (kazuho) [inline draft-kazuho-httpbis-priority.md]: Maybe better to say "encoded using Structured Headers"?

File: draft-kazuho-httpbis-priority.md [hunk 3]
```diff
@@ -488,6 +502,13 @@ Element ID is interpreted as a Push ID.
 Empty:
 : A seven-bit field that has no semantic value.
 
+Prioritized Element ID:
+: The stream ID or push ID that is the target of the priority 
+update.
+
+Priority Field Value:
+: The priority update value in ASCII text.
```
Comment (kazuho) [inline draft-kazuho-httpbis-priority.md]: Here too.

**Review by ianswett — APPROVED** [review]
One nit on indent, but LG

File: draft-kazuho-httpbis-priority.md (hunk 1 above)
Comment (kazuho) [reply draft-kazuho-httpbis-priority.md]: Done in 5bb07f7.

File: draft-kazuho-httpbis-priority.md (hunk 2 above)
Comment (kazuho) [reply draft-kazuho-httpbis-priority.md]: Done in 9ed8908.

**Pull request merged**